_posts/2023-10-30-announcement-bnd-pde-cooperation.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: Announcement - bnd / bndtools and PDE cooperation
+description: PDE and bndtools unite to make OSGi development better 
+comments: true
+author: Christoph Rüger
+---
+
+At the recent EclipseCon 2023 conference, the communities behind **Eclipse PDE** and **bnd / bndtools** decided to better integrate their OSGi development tools.
+Historically, these groups were divided in their approach. Now, there's a push for improved compatibility.
```

Comment:
"...improved compatibility and collaboration"

Reply:
Done